extensions/amp-analytics/0.1/requests.js

```diff
@@ -110,8 +110,11 @@ export class RequestHandler {
    * @param {!Object<string, *>} dynamicBindings A mapping of variables to
    *     stringable values. For example, values could be strings, functions that
    *     return strings, promises, etc.
+   * @param {Document=} opt_contextDoc Used if any of the expansion variables
```

Comment:
Do you want this optional param to be non-null when passed? If so, use `!Document=`, and same below.

Reply:
Sure, I can add tests. Will also make the change you recommend here.